packages/mistral/conf/mistral.conf

```diff
@@ -840,3 +843,4 @@
 
 # Enables user authentication in pecan. (boolean value)
 #auth_enable = true
+auth_enable = false
```

Comment:
Guys, keep in mind: I have no idea what I'm doing, since it's mistral.
For example, how bad is this value for default installation?